azure-pipelines.yml

```
@@ -8,7 +8,7 @@ variables:
 jobs:
 - job: 'Default'
   pool:
-    vmImage: 'vs2017-win2016'
+    vmImage: 'windows-latest'
```

```suggestion
    vmImage: 'windows-2019'
```

How backward compatible is a build on windows-2022 going to be? SHould we stick to 2019?

---

I'm not sure. I wasn't too worried as this is just for running the tests, not for distributing built wheels. If it breaks when latest switches to 2022 then we can always pin to 2019 at that time.

I did check what NumPy is using and they currently have `windows-latest`.

---

oh ok. for tests then it is fine.